share/spice/playing_cards/playing_cards.js

```diff
@@ -0,0 +1,29 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_playing_cards = function(api_result){
+        if(api_result.error || !api_result.cards) {
+            return Spice.failed('playing_cards');
+        }
+        
+        Spice.add({
+            id: "playing_cards",
+            name: "Deck of Cards",
+            data: api_result.cards,
+            meta: {
+                sourceName: "DeckOfCards",
+                sourceUrl: 'http://deckofcardsapi.com'
+            },
+            normalize: function(item) {
+                return {
+                    image: item.image
+                };
+            },
+            templates: {
+                group: 'movies',
```

Comment:
Latest snag: I've made the other changes (smaller ```img``` of 12em height, tile with auto width, transparent border and background), and now need to set the img wrap to have an ```auto``` height. Can I have some help? Whatever I *am* trying isn't working.

Here's the [screenshot](https://goo.gl/8pWCq7), and here's the css:
```css
.zci--playing_cards .tile--playing_cards {
    width: auto;
    background-color: transparent;
    border: transparent;
}

.tile--playing_cards .tile__media__img {
    height: 12em;
}
```